### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by myelin0**
## In this branch I add the views for controllers actions:

- Use preprocessed HTML file with embedded Ruby code.
- Use layouts and templates for shared content.
- Pass linters with Rubocop.

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,17 @@
...
+        </div>
+    </div>
+    <div class="user_page_bio">
+        <span class="bio_header">Bio</span>
+        <p class="user_bio"><%= @user.bio %> </p>
+    </div>
+    <div class="user_page_posts">
+        <%= render partial: "posts" %>
+    </div>
+    <button class="see_all_posts_btn">See all posts</button>
```
Comment (SajjadAhmad14) [inline app/views/users/show.html.erb]: The `See all Posts` link should open the `user's post index` page. It's just a button now with no link. Kindly add a link to the `user's post index` page here.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Emi, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app/views/users/show.html.erb (hunk 1 above)
Comment (myelin0) [reply app/views/users/show.html.erb]: Thanks for the review. I added the link to the user`s post index now.

**Review by KDlamini — APPROVED** [review]
### Hi @myelin0  🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the required changes. 🎊 🥳 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

## HIGHLIGHTS 🌟 

✔️ Blog views implemented correctly
✔️ Correct PR title and description
✔️ Correct use of Gitflow
✔️ No linter errors

# Status: Approved ✔️ 

simon

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**